l1-contracts/test/NewOutbox.t.sol

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Aztec Labs.
+pragma solidity >=0.8.18;
+
+import {Test} from "forge-std/Test.sol";
+import {NewOutbox} from "../src/core/messagebridge/NewOutbox.sol";
+import {INewOutbox} from "../src/core/interfaces/messagebridge/INewOutbox.sol";
+import {Errors} from "../src/core/libraries/Errors.sol";
+import {DataStructures} from "../src/core/libraries/DataStructures.sol";
+import {Merkle} from "../src/core/libraries/Merkle.sol";
+import {Hash} from "../src/core/libraries/Hash.sol";
+import {NaiveMerkle} from "./merkle/Naive.sol";
+
+contract NewOutboxTest is Test {
+  using Hash for DataStructures.L2ToL1Msg;
+
+  address internal constant _STATE_TRANSITIONER = address(0x42069123);
+  address internal constant _NOT_STATE_TRANSITIONER = address(0x69);
+  address internal constant _NOT_RECIPIENT = address(0x420);
+  uint256 internal constant _DEFAULT_TREE_HEIGHT = 2;
+  uint256 internal constant _VERSION = 0;
```

Comment:
We don't currently use the underscore for constants in our codebase so would keep that consistent.